packages/config/src/forkConfig/index.ts

```diff
@@ -138,15 +138,32 @@ export function createForkConfig(config: ChainConfig): ForkConfig {
       }
       return sszTypesFor(forkName);
     },
-    getMaxBlobsPerBlock(fork: ForkName): number {
-      switch (fork) {
-        case ForkName.fulu:
-          return config.MAX_BLOBS_PER_BLOCK_FULU;
-        case ForkName.electra:
-          return config.MAX_BLOBS_PER_BLOCK_ELECTRA;
-        default:
-          return config.MAX_BLOBS_PER_BLOCK;
+    getMaxBlobsPerBlock(epoch: Epoch): number {
+      if (config.BLOB_SCHEDULE.length === 0) {
+        throw Error("Attempt to get MAX_BLOBS_PER_BLOCK from empty BLOB_SCHEDULE");
       }
+
+      const minEpoch = Math.min(...config.BLOB_SCHEDULE.map((e) => e.EPOCH));
+      if (epoch < minEpoch) {
```

Comment:
This change is interfering will kurtosis configs that Barnabus is running for devnet-0 prep.

```
network_params:
  fulu_fork_epoch: 1
  bpo_1_epoch: 1
  bpo_1_max_blobs: 9
  bpo_1_target_blobs: 6
  bpo_2_epoch: 2
  bpo_2_max_blobs: 18
  bpo_2_target_blobs: 12
  bpo_3_epoch: 3
  bpo_3_max_blobs: 9
  bpo_3_target_blobs: 6
  bpo_4_epoch: 4
  bpo_4_max_blobs: 18
  bpo_4_target_blobs: 12
  bpo_5_epoch: 5
  bpo_5_max_blobs: 9
  bpo_5_target_blobs: 6
```

This error will trigger there is no blob schedule for Electra (epoch 0). Will either need to remove this min epoch requirement, or communicate our expectation (and our interpretation of blob schedule) to the interop people